bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/devices/AbstractPowerSwitchHandler.java

```diff
@@ -74,8 +74,8 @@ public void handleCommand(ChannelUID channelUID, Command command) {
 
         switch (channelUID.getId()) {
             case CHANNEL_POWER_SWITCH:
-                if (command instanceof OnOffType) {
-                    updatePowerSwitchState((OnOffType) command);
+                if (command instanceof OnOffType onOffType) {
```

Comment:
Minor, perhaps this naming would be slightly easier to understand?
```suggestion
                if (command instanceof OnOffType onOffCommand) {
```

If you agree, consider all others as well.

Reply:
Good idea 👍 I changed it consistently in all places (4 classes).